webrender/src/device/gfx/device.rs

```diff
@@ -3354,13 +3200,12 @@ impl<B: hal::Backend> Device<B> {
                     DeviceIntSize::new(self.viewport.rect.w as _, self.viewport.rect.h as _),
                 )
             };
-            if rect.size.width > target_rect.size.width
-                || rect.size.height > target_rect.size.height
-            {
-                // This can happen, when we resize
-                self.clear_target_image(color, depth);
-            } else if rect == target_rect {
-                self.clear_target_image(color, depth);
+            rect.size.width = rect.size.width.min(target_rect.size.width);
+            rect.size.height = rect.size.height.min(target_rect.size.height);
+            if !self.inside_renderpass {
```

Comment:
as I mentioned before, we can just totally ignore the rect and clear the whole image for now (without starting/ending a render pass)

Reply:
As I've found there are places where we need those rect clears, and can't clear the whole image, otherwise the rendering result will be wrong.